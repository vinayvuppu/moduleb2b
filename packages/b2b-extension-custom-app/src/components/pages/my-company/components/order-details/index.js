export { default } from './order-details';
