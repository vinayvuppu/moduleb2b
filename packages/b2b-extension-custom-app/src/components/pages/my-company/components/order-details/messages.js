import { defineMessages } from 'react-intl';

export default defineMessages({
  title: {
    id: 'OrderDetails.title',
    defaultMessage: 'Order',
  },
  backToList: {
    id: 'OrderDetails.backToList',
    defaultMessage: 'To order list',
  },
  summaryTitle: {
    id: 'OrderDetails.summaryTitle',
    defaultMessage: 'Order summary',
  },
  orderStateLabel: {
    id: 'OrderDetails.orderStateLabel',
    defaultMessage: 'Order state',
  },
  paymentStateLabel: {
    id: 'OrderDetails.paymentStateLabel',
    defaultMessage: 'Payment state',
  },
  shipmentStateLabel: {
    id: 'OrderDetails.shipmentStateLabel',
    defaultMessage: 'Shipment state',
  },
  customerTitle: {
    id: 'OrderDetails.customerTitle',
    defaultMessage: 'Employee',
  },
  customerEmailLabel: {
    id: 'OrderDetails.customerEmailLabel',
    defaultMessage: 'Employee email: ',
  },
  orderItemsTitle: {
    id: 'OrderDetails.orderItemsTitle',
    defaultMessage: 'Order items',
  },
  productColumn: {
    id: 'OrderDetails.productColumn',
    defaultMessage: 'Product',
  },
  quantityColumn: {
    id: 'OrderDetails.quantityColumn',
    defaultMessage: 'Qty',
  },
  taxColumn: {
    id: 'OrderDetails.taxColumn',
    defaultMessage: 'Tax',
  },
  unitPriceColumn: {
    id: 'OrderDetails.unitPriceColumn',
    defaultMessage: 'Unit price',
  },
  subtotalColumn: {
    id: 'OrderDetails.subtotalColumn',
    defaultMessage: 'Subtotal',
  },
  totalColumn: {
    id: 'OrderDetails.totalColumn',
    defaultMessage: 'Total',
  },
  totalLabel: {
    id: 'OrderDetails.totalLabel',
    defaultMessage: 'Total',
  },
  subTotalWithoutDiscountLabel: {
    id: 'OrderDetails.subTotalWithoutDiscountLabel',
    defaultMessage: 'Subtotal without discount',
  },
  subTotalGrossLabel: {
    id: 'OrderDetails.subTotalGrossLabel',
    defaultMessage: 'Subtotal (gross)',
  },
  shippingInclLabel: {
    id: 'OrderDetails.shippingInclLabel',
    defaultMessage: 'Shipping (incl. tax):',
  },
  totalGross: {
    id: 'OrderDetails.totalGross',
    defaultMessage: 'Total (gross)',
  },
});
