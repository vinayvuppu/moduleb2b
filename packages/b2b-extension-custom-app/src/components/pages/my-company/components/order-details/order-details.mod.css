.card {
  min-width: 410px;
  max-width: 550px;
}

.order-items {
  width: 100%;
}

.card-prices > div > div > :last-child {
  border-top: 1px solid #ccc;
  padding-top: 0.5rem;
}
