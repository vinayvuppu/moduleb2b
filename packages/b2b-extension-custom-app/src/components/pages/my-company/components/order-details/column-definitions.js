import messages from './messages';

export default formatMessage => [
  {
    key: 'product',
    label: formatMessage(messages.productColumn),
    flexGrow: 1,
  },
  {
    key: 'unitPrice',
    label: formatMessage(messages.unitPriceColumn),
  },
  {
    key: 'quantity',
    label: formatMessage(messages.quantityColumn),
  },
  {
    key: 'subtotal',
    label: formatMessage(messages.subtotalColumn),
  },
  {
    key: 'taxRate',
    label: formatMessage(messages.taxColumn),
  },
  {
    key: 'total',
    label: formatMessage(messages.totalColumn),
  },
];
