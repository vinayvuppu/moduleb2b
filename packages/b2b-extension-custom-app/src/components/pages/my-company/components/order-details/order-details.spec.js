import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import { IntlProvider } from 'react-intl';
import { OrderDetails } from './order-details';

const order = {
  orderState: 'Confirmed',
  paymentState: 'Pending',
  shipmentState: 'Pending',
  shippingInfo: {
    shippingMethodName: 'Express US',
    taxRate: {
      name: '10% incl.',
      amount: 0.1,
    },
    taxedPrice: {
      totalNet: {
        type: 'centPrecision',
        currencyCode: 'USD',
        centAmount: 909,
        fractionDigits: 2,
      },
      totalGross: {
        type: 'centPrecision',
        currencyCode: 'USD',
        centAmount: 1000,
        fractionDigits: 2,
      },
    },
  },
  taxedPrice: {
    totalNet: {
      type: 'centPrecision',
      currencyCode: 'USD',
      centAmount: 13182,
      fractionDigits: 2,
    },
    totalGross: {
      type: 'centPrecision',
      currencyCode: 'USD',
      centAmount: 14500,
      fractionDigits: 2,
    },
    taxPortions: [
      {
        name: '10% incl.',
        amount: {
          type: 'centPrecision',
          currencyCode: 'USD',
          centAmount: 1318,
          fractionDigits: 2,
        },
      },
    ],
  },
  employee: {
    id: '05502506-d831-49de-ab16-9026c319c07f',
    email: 'florian@devgurus.io',
  },
  totalPrice: {
    type: 'centPrecision',
    currencyCode: 'USD',
    centAmount: 10300,
    fractionDigits: 2,
  },
  shippingAddress: {
    additionalStreetInfo: null,
    country: 'US',
    additionalAddressInfo: null,
  },
  billingAddress: {
    additionalStreetInfo: null,
    country: 'US',
    additionalAddressInfo: null,
  },
  lineItems: [
    {
      id: '1635aa91-1b6d-489e-a3c1-4dd2cc8608c6',
      name: 'Top Jucca black',
      quantity: 1,
      custom: {
        customFieldsRaw: [
          {
            name: 'originalPrice',
            value: {
              fractionDigits: 2,
              centAmount: 13500,
              currencyCode: 'USD',
              type: 'centPrecision',
            },
          },
        ],
      },
      totalPrice: {
        centAmount: 10000,
      },
      taxRate: {
        amount: 0.1,
        subRates: [],
      },
      taxedPrice: {
        totalNet: {
          type: 'centPrecision',
          currencyCode: 'USD',
          centAmount: 9091,
          fractionDigits: 2,
        },
        totalGross: {
          type: 'centPrecision',
          currencyCode: 'USD',
          centAmount: 10000,
          fractionDigits: 2,
        },
      },
      variant: {
        sku: 'M0E20000000EC8N',
        images: [
          {
            url:
              'https://s3-eu-west-1.amazonaws.com/commercetools-maximilian/products/081138_1_large.jpg',
          },
        ],
      },
    },
  ],
};

describe('Order details component', () => {
  test('render snapshot', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <IntlProvider locale="en">
          <OrderDetails order={order} backUrl="www.foo.com" />
        </IntlProvider>
      </MemoryRouter>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
