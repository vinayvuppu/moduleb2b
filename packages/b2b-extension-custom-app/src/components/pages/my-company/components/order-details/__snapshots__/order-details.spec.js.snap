// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Order details component render snapshot 1`] = `
<DocumentFragment>
  <div
    class="view"
  >
    <div
      class="header-color-neutral header-color-with-bottom-line"
    >
      <div
        class="details"
      >
        <div
          class="info"
        >
          <a
            aria-label="To order list"
            class="css-1nkn7kf-StyledExternalLink-hoverStyles e1tn0gr10"
            data-track-component="LinkButton"
            href="/www.foo.com"
            label="To order list"
          >
            <span
              class="css-18wydpa-getStyles"
            >
              <svg
                class="css-vg0gg2-getIconStyles"
                color="primary"
                height="24"
                viewBox="0 0 24 24"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill-rule="evenodd"
                  id="list_react_svg__Icons"
                  stroke="none"
                  stroke-width="1"
                >
                  <g
                    fill="#000"
                    id="list_react_svg__MC-icon-set"
                    transform="translate(-24 -600)"
                  >
                    <g
                      id="list_react_svg__Grid-display"
                      transform="translate(24 600)"
                    >
                      <g
                        id="list_react_svg__List"
                      >
                        <path
                          d="M5.248 9.837a.31.31 0 01.226.094.304.304 0 01.095.223v1.905c0 .086-.031.16-.095.223a.31.31 0 01-.226.094H3.321a.311.311 0 01-.226-.094.303.303 0 01-.095-.223v-1.905c0-.086.032-.16.095-.223a.31.31 0 01.226-.094h1.927zm0 7.617a.31.31 0 01.226.094.303.303 0 01.095.224v1.904c0 .086-.031.16-.095.223a.31.31 0 01-.226.094H3.321a.31.31 0 01-.226-.094.303.303 0 01-.095-.223v-1.904c0-.086.032-.16.095-.224a.31.31 0 01.226-.094h1.927zm0-3.808a.31.31 0 01.226.094.304.304 0 01.095.223v1.904c0 .086-.031.16-.095.223a.31.31 0 01-.226.095H3.321a.31.31 0 01-.226-.095.303.303 0 01-.095-.223v-1.904c0-.086.032-.16.095-.223a.311.311 0 01.226-.094h1.927zm0-7.618c.087 0 .163.032.226.095a.303.303 0 01.095.223V8.25c0 .086-.031.16-.095.223a.31.31 0 01-.226.094H3.321a.31.31 0 01-.226-.094A.303.303 0 013 8.25V6.346c0-.086.032-.16.095-.223a.31.31 0 01.226-.095h1.927zm15.418 11.426c.087 0 .162.032.226.094a.303.303 0 01.095.224v1.904c0 .086-.032.16-.095.223a.31.31 0 01-.226.094H7.176a.31.31 0 01-.226-.094.303.303 0 01-.096-.223v-1.904c0-.086.032-.16.096-.224a.31.31 0 01.226-.094h13.49zm0-3.808c.087 0 .162.031.226.094a.304.304 0 01.095.223v1.904c0 .086-.032.16-.095.223a.31.31 0 01-.226.095H7.176a.31.31 0 01-.226-.095.303.303 0 01-.096-.223v-1.904c0-.086.032-.16.096-.223a.311.311 0 01.226-.094h13.49zm.226-7.523a.304.304 0 01.095.223V8.25c0 .086-.032.16-.095.223a.31.31 0 01-.226.094H7.176a.31.31 0 01-.226-.094.303.303 0 01-.096-.223V6.346c0-.086.032-.16.096-.223a.31.31 0 01.226-.095h13.49c.087 0 .162.032.226.095zm-.226 3.714a.31.31 0 01.226.094.304.304 0 01.095.223v1.905c0 .086-.032.16-.095.223a.31.31 0 01-.226.094H7.176a.311.311 0 01-.226-.094.303.303 0 01-.096-.223v-1.905c0-.086.032-.16.096-.223a.31.31 0 01.226-.094h13.49z"
                          id="list_react_svg__shape"
                        />
                      </g>
                    </g>
                  </g>
                </g>
              </svg>
              <span
                class="css-f4iurr-bodyStyles"
              >
                To order list
              </span>
            </span>
          </a>
          <div
            class="title-container"
          >
            <h2
              class="css-1d7aioo-headlineStyles"
            >
              Order
            </h2>
          </div>
          <div
            class="tabs"
          >
            <ul
              class="tabs-list"
            />
          </div>
        </div>
        <div
          class="commands"
        />
      </div>
    </div>
    <div
      class="css-1um456l-Inset"
    >
      <div
        class="css-z1wx52-getStyles"
      >
        <div
          class="css-y8qm83-Horizontal-getConstraintStyles-Horizontal"
        >
          <div
            class="css-1exe63e-baseContainerStyles-getThemeStyle-CollapsiblePanel"
          >
            <div
              aria-controls="panel-content-1"
              aria-disabled="false"
              aria-expanded="true"
              aria-label=""
              class="etpmm8q0 css-wtj13x-Button-normalizedButtonStyles-AccessibleButton-HeaderContainer-baseStyles-getHeaderContainerStyles-getThemeStyle-CollapsiblePanel e1hlacz70"
              id="1"
              role="button"
              tabindex="0"
            >
              <span
                class="css-16vcylu-getStyles"
              >
                <div
                  class="css-2yi65l-HeaderIcon-HeaderIcon"
                >
                  <svg
                    class="css-msim5y-getIconStyles"
                    color="solid"
                    height="24"
                    viewBox="0 0 24 24"
                    width="24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <g
                      fill-rule="evenodd"
                      id="angle-down_react_svg__Icons"
                      stroke="none"
                      stroke-width="1"
                    >
                      <g
                        fill="#000"
                        id="angle-down_react_svg__MC-icon-set"
                        transform="translate(-168 -888)"
                      >
                        <g
                          id="angle-down_react_svg__Directions"
                          transform="translate(24 888)"
                        >
                          <g
                            id="angle-down_react_svg__Angle-Down"
                            transform="translate(144)"
                          >
                            <path
                              d="M20.816 8.115l-.902-.909a.567.567 0 00-.415-.182.567.567 0 00-.414.182l-7.087 7.147L4.91 7.206a.567.567 0 00-.414-.181.567.567 0 00-.415.181l-.902.91a.576.576 0 000 .836l8.403 8.474c.12.12.259.182.415.182.156 0 .294-.061.414-.182l8.404-8.474a.577.577 0 000-.837z"
                              id="angle-down_react_svg__shape"
                            />
                          </g>
                        </g>
                      </g>
                    </g>
                  </svg>
                </div>
                <span
                  class="css-16lu50f-getStyles"
                >
                  <h4
                    class="css-1oq94km-subheadlineStyles"
                  >
                    Order summary
                  </h4>
                </span>
              </span>
            </div>
            <div
              style="height: auto;"
            >
              <div
                class="css-1t8jyu-SectionWrapper erh4hih0"
              >
                <div
                  class="css-1um456l-Inset"
                >
                  <div
                    aria-hidden="false"
                    class="css-ps2rzv-SectionContent erh4hih1"
                    id="panel-content-1"
                  >
                    <span
                      class="css-79qb0q-getStyles"
                    >
                      <div
                        class="card css-ngf9uq-Card"
                      >
                        <div
                          class="css-z1wx52-getStyles"
                        >
                          <span
                            class="css-1t8v07g-getStyles"
                          >
                            <p
                              class="css-17wfq1-bodyStyles"
                            >
                              Order state
                            </p>
                            <p
                              class="css-f4iurr-bodyStyles"
                            >
                              Confirmed
                            </p>
                          </span>
                          <span
                            class="css-1t8v07g-getStyles"
                          >
                            <p
                              class="css-17wfq1-bodyStyles"
                            >
                              Payment state
                            </p>
                            <p
                              class="css-f4iurr-bodyStyles"
                            >
                              Pending
                            </p>
                          </span>
                          <span
                            class="css-1t8v07g-getStyles"
                          >
                            <p
                              class="css-17wfq1-bodyStyles"
                            >
                              Shipment state
                            </p>
                            <p
                              class="css-f4iurr-bodyStyles"
                            >
                              Pending
                            </p>
                          </span>
                        </div>
                      </div>
                      <div
                        class="card card-prices css-ngf9uq-Card"
                      >
                        <div
                          class="css-z1wx52-getStyles"
                        >
                          <span
                            class="css-1t8v07g-getStyles"
                          >
                            <p
                              class="css-17wfq1-bodyStyles"
                            >
                              Subtotal without discount
                            </p>
                            <p
                              class="css-17wfq1-bodyStyles"
                            >
                              $122.73
                            </p>
                          </span>
                          <span
                            class="css-1t8v07g-getStyles"
                          >
                            <p
                              class="css-118j872-bodyStyles"
                            >
                              10% incl.
                            </p>
                            <p
                              class="css-118j872-bodyStyles"
                            >
                              + $13.18
                            </p>
                          </span>
                          <span
                            class="css-1t8v07g-getStyles"
                          >
                            <p
                              class="css-17wfq1-bodyStyles"
                            >
                              Subtotal (gross)
                            </p>
                            <p
                              class="css-17wfq1-bodyStyles"
                            >
                              $135.00
                            </p>
                          </span>
                          <span
                            class="css-1t8v07g-getStyles"
                          >
                            <p
                              class="css-118j872-bodyStyles"
                            >
                              Shipping (incl. tax): Express US
                            </p>
                            <p
                              class="css-118j872-bodyStyles"
                            >
                              + $10.00
                            </p>
                          </span>
                          <span
                            class="css-1t8v07g-getStyles"
                          >
                            <h4
                              class="css-ig3dml-subheadlineStyles"
                            >
                              Total (gross)
                            </h4>
                            <h4
                              class="css-ig3dml-subheadlineStyles"
                            >
                              $103.00
                            </h4>
                          </span>
                        </div>
                      </div>
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="css-y8qm83-Horizontal-getConstraintStyles-Horizontal"
        >
          <div
            class="css-1exe63e-baseContainerStyles-getThemeStyle-CollapsiblePanel"
          >
            <div
              aria-controls="panel-content-1"
              aria-disabled="false"
              aria-expanded="true"
              aria-label=""
              class="etpmm8q0 css-wtj13x-Button-normalizedButtonStyles-AccessibleButton-HeaderContainer-baseStyles-getHeaderContainerStyles-getThemeStyle-CollapsiblePanel e1hlacz70"
              id="1"
              role="button"
              tabindex="0"
            >
              <span
                class="css-16vcylu-getStyles"
              >
                <div
                  class="css-2yi65l-HeaderIcon-HeaderIcon"
                >
                  <svg
                    class="css-msim5y-getIconStyles"
                    color="solid"
                    height="24"
                    viewBox="0 0 24 24"
                    width="24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <g
                      fill-rule="evenodd"
                      id="angle-down_react_svg__Icons"
                      stroke="none"
                      stroke-width="1"
                    >
                      <g
                        fill="#000"
                        id="angle-down_react_svg__MC-icon-set"
                        transform="translate(-168 -888)"
                      >
                        <g
                          id="angle-down_react_svg__Directions"
                          transform="translate(24 888)"
                        >
                          <g
                            id="angle-down_react_svg__Angle-Down"
                            transform="translate(144)"
                          >
                            <path
                              d="M20.816 8.115l-.902-.909a.567.567 0 00-.415-.182.567.567 0 00-.414.182l-7.087 7.147L4.91 7.206a.567.567 0 00-.414-.181.567.567 0 00-.415.181l-.902.91a.576.576 0 000 .836l8.403 8.474c.12.12.259.182.415.182.156 0 .294-.061.414-.182l8.404-8.474a.577.577 0 000-.837z"
                              id="angle-down_react_svg__shape"
                            />
                          </g>
                        </g>
                      </g>
                    </g>
                  </svg>
                </div>
                <span
                  class="css-16lu50f-getStyles"
                >
                  <h4
                    class="css-1oq94km-subheadlineStyles"
                  >
                    Employee
                  </h4>
                </span>
              </span>
            </div>
            <div
              style="height: auto;"
            >
              <div
                class="css-1t8jyu-SectionWrapper erh4hih0"
              >
                <div
                  class="css-1um456l-Inset"
                >
                  <div
                    aria-hidden="false"
                    class="css-ps2rzv-SectionContent erh4hih1"
                    id="panel-content-1"
                  >
                    <div
                      class="css-z1wx52-getStyles"
                    >
                      <span
                        class="css-16vcylu-getStyles"
                      >
                        <small
                          class="css-umditc-detailStyles"
                        >
                          Employee email:  florian@devgurus.io
                        </small>
                        <svg
                          class="css-19kq3xe-getIconStyles"
                          height="24"
                          viewBox="0 0 24 24"
                          width="24"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <g
                            fill="none"
                            fill-rule="evenodd"
                            id="mail_react_svg__Icons"
                            stroke="none"
                            stroke-width="1"
                          >
                            <g
                              fill="#000"
                              fill-rule="nonzero"
                              id="mail_react_svg__MC-icon-set"
                              transform="translate(-96 -232)"
                            >
                              <g
                                id="mail_react_svg__CRUD"
                                transform="translate(24 152)"
                              >
                                <path
                                  d="M12 13.312L3 6h18l-9 7.312zm-2.244-1.046L3 6.926V17.41l6.756-5.144zm2.398 1.752c-.112.092-.194.093-.308 0l-1.599-1.335-7.221 5.653h17.948l-7.221-5.653-1.599 1.335zm2.112-1.73l6.757-5.34v10.485l-6.757-5.145z"
                                  id="mail_react_svg__Mail"
                                  transform="translate(72 80)"
                                />
                              </g>
                            </g>
                          </g>
                        </svg>
                      </span>
                      <span
                        class="css-79qb0q-getStyles"
                      >
                        <div
                          class="card css-ngf9uq-Card"
                        >
                          <div
                            class="css-z1wx52-getStyles"
                          >
                            <span
                              class="css-79qb0q-getStyles"
                            >
                              <h4
                                class="css-78wsr5-subheadlineStyles"
                              >
                                Shipping to
                              </h4>
                              <svg
                                class="css-19kq3xe-getIconStyles"
                                height="24"
                                viewBox="0 0 24 24"
                                width="24"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <g
                                  fill-rule="evenodd"
                                  id="truck_react_svg__Icons"
                                  stroke="none"
                                  stroke-width="1"
                                >
                                  <g
                                    fill="#000"
                                    id="truck_react_svg__MC-icon-set"
                                    transform="translate(-96 -408)"
                                  >
                                    <g
                                      id="truck_react_svg__Switch-states"
                                      transform="translate(24 408)"
                                    >
                                      <g
                                        id="truck_react_svg__Truck"
                                        transform="translate(72)"
                                      >
                                        <path
                                          d="M20.19 14.308a2.9 2.9 0 00-5.535 0h-1.27V7.385l2.43-.012c1.914-.008 3.149 1.082 2.762 2.435l1.59 1.061c.454.304.833 1.008.833 1.565v1.874h-.81zm-9.75 0a2.9 2.9 0 00-5.535 0H3v-4.3A4.01 4.01 0 016.998 6h5.694v8.308h-2.251zm-2.767 3.115a2.25 2.25 0 110-4.5 2.25 2.25 0 010 4.5zm9.75 0a2.25 2.25 0 110-4.5 2.25 2.25 0 010 4.5z"
                                          id="truck_react_svg__shape"
                                        />
                                      </g>
                                    </g>
                                  </g>
                                </g>
                              </svg>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <small
                                class="css-umditc-detailStyles"
                              >
                                Name:
                              </small>
                              <small
                                class="css-umditc-detailStyles"
                              >
                                -
                              </small>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <small
                                class="css-umditc-detailStyles"
                              >
                                Address:
                              </small>
                              <small
                                class="css-umditc-detailStyles"
                              >
                                -
                              </small>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <small
                                class="css-umditc-detailStyles"
                              >
                                Addition street information:
                              </small>
                              <small
                                class="css-umditc-detailStyles"
                              >
                                -
                              </small>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <small
                                class="css-umditc-detailStyles"
                              >
                                City:
                              </small>
                              <small
                                class="css-umditc-detailStyles"
                              >
                                -
                              </small>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <small
                                class="css-umditc-detailStyles"
                              >
                                Postal code:
                              </small>
                              <small
                                class="css-umditc-detailStyles"
                              >
                                -
                              </small>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <small
                                class="css-umditc-detailStyles"
                              >
                                Country:
                              </small>
                              <small
                                class="css-umditc-detailStyles"
                              >
                                US
                              </small>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <p
                                class="css-f4iurr-bodyStyles"
                              >
                                Addition address information:
                              </p>
                              <p
                                class="css-f4iurr-bodyStyles"
                              >
                                -
                              </p>
                            </span>
                          </div>
                        </div>
                        <div
                          class="card css-ngf9uq-Card"
                        >
                          <div
                            class="css-z1wx52-getStyles"
                          >
                            <span
                              class="css-79qb0q-getStyles"
                            >
                              <h4
                                class="css-78wsr5-subheadlineStyles"
                              >
                                Billing to
                              </h4>
                              <svg
                                class="css-19kq3xe-getIconStyles"
                                height="24"
                                viewBox="0 0 24 24"
                                width="24"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <g
                                  fill-rule="evenodd"
                                  id="paper-bill-inverted_react_svg__Icons"
                                  stroke="none"
                                  stroke-width="1"
                                >
                                  <g
                                    fill="#000"
                                    id="paper-bill-inverted_react_svg__MC-icon-set"
                                    transform="translate(-24 -408)"
                                  >
                                    <g
                                      id="paper-bill-inverted_react_svg__Switch-states"
                                      transform="translate(24 408)"
                                    >
                                      <g
                                        id="paper-bill-inverted_react_svg__Paper-Bill-Inverted"
                                      >
                                        <path
                                          d="M16.324 19.7h-.002v.006L14.913 21l-1.42-1.256V19.7h-.002v.006L12.083 21l-1.42-1.256V19.7h-.002v.006L9.25 21l-1.42-1.256V19.7H7.83v.006L6.42 21 5 19.744V4.342L6.42 3l1.408 1.341V4.4h.003v-.058L9.251 3l1.408 1.341V4.4h.003v-.058L12.082 3l1.407 1.341V4.4h.003v-.058L14.913 3l1.407 1.341V4.4h.004v-.058L17.744 3l1.407 1.341.002 15.365L17.744 21l-1.42-1.256V19.7zM6.857 8.225c0 .258.22.481.49.481h8.64c.272 0 .491-.215.491-.481a.49.49 0 00-.49-.481h-8.64a.485.485 0 00-.49.48zm0 2.886c0 .258.22.481.49.481h8.64c.272 0 .491-.215.491-.48a.49.49 0 00-.49-.482h-8.64a.485.485 0 00-.49.481zm.284 2.887c0 .257.212.48.473.48h4.828a.482.482 0 000-.961H7.614a.482.482 0 00-.473.48z"
                                          id="paper-bill-inverted_react_svg__shape"
                                        />
                                      </g>
                                    </g>
                                  </g>
                                </g>
                              </svg>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <small
                                class="css-umditc-detailStyles"
                              >
                                Name:
                              </small>
                              <small
                                class="css-umditc-detailStyles"
                              >
                                -
                              </small>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <small
                                class="css-umditc-detailStyles"
                              >
                                Address:
                              </small>
                              <small
                                class="css-umditc-detailStyles"
                              >
                                -
                              </small>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <small
                                class="css-umditc-detailStyles"
                              >
                                Addition street information:
                              </small>
                              <small
                                class="css-umditc-detailStyles"
                              >
                                -
                              </small>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <small
                                class="css-umditc-detailStyles"
                              >
                                City:
                              </small>
                              <small
                                class="css-umditc-detailStyles"
                              >
                                -
                              </small>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <small
                                class="css-umditc-detailStyles"
                              >
                                Postal code:
                              </small>
                              <small
                                class="css-umditc-detailStyles"
                              >
                                -
                              </small>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <small
                                class="css-umditc-detailStyles"
                              >
                                Country:
                              </small>
                              <small
                                class="css-umditc-detailStyles"
                              >
                                US
                              </small>
                            </span>
                            <span
                              class="css-1t8v07g-getStyles"
                            >
                              <p
                                class="css-f4iurr-bodyStyles"
                              >
                                Addition address information:
                              </p>
                              <p
                                class="css-f4iurr-bodyStyles"
                              >
                                -
                              </p>
                            </span>
                          </div>
                        </div>
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="css-y8qm83-Horizontal-getConstraintStyles-Horizontal"
        >
          <div
            class="css-1exe63e-baseContainerStyles-getThemeStyle-CollapsiblePanel"
          >
            <div
              aria-controls="panel-content-1"
              aria-disabled="false"
              aria-expanded="true"
              aria-label=""
              class="etpmm8q0 css-wtj13x-Button-normalizedButtonStyles-AccessibleButton-HeaderContainer-baseStyles-getHeaderContainerStyles-getThemeStyle-CollapsiblePanel e1hlacz70"
              id="1"
              role="button"
              tabindex="0"
            >
              <span
                class="css-16vcylu-getStyles"
              >
                <div
                  class="css-2yi65l-HeaderIcon-HeaderIcon"
                >
                  <svg
                    class="css-msim5y-getIconStyles"
                    color="solid"
                    height="24"
                    viewBox="0 0 24 24"
                    width="24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <g
                      fill-rule="evenodd"
                      id="angle-down_react_svg__Icons"
                      stroke="none"
                      stroke-width="1"
                    >
                      <g
                        fill="#000"
                        id="angle-down_react_svg__MC-icon-set"
                        transform="translate(-168 -888)"
                      >
                        <g
                          id="angle-down_react_svg__Directions"
                          transform="translate(24 888)"
                        >
                          <g
                            id="angle-down_react_svg__Angle-Down"
                            transform="translate(144)"
                          >
                            <path
                              d="M20.816 8.115l-.902-.909a.567.567 0 00-.415-.182.567.567 0 00-.414.182l-7.087 7.147L4.91 7.206a.567.567 0 00-.414-.181.567.567 0 00-.415.181l-.902.91a.576.576 0 000 .836l8.403 8.474c.12.12.259.182.415.182.156 0 .294-.061.414-.182l8.404-8.474a.577.577 0 000-.837z"
                              id="angle-down_react_svg__shape"
                            />
                          </g>
                        </g>
                      </g>
                    </g>
                  </svg>
                </div>
                <span
                  class="css-16lu50f-getStyles"
                >
                  <h4
                    class="css-1oq94km-subheadlineStyles"
                  >
                    Order items
                  </h4>
                </span>
              </span>
            </div>
            <div
              style="height: auto;"
            >
              <div
                class="css-1t8jyu-SectionWrapper erh4hih0"
              >
                <div
                  class="css-1um456l-Inset"
                >
                  <div
                    aria-hidden="false"
                    class="css-ps2rzv-SectionContent erh4hih1"
                    id="panel-content-1"
                  >
                    <div
                      class="order-items"
                    >
                      <div
                        class="css-z1wx52-getStyles"
                      >
                        <div
                          class="css-1r22j04-Table"
                        >
                          <div
                            style="overflow: visible; width: 0px;"
                          >
                            <div
                              style="width: 0px;"
                            >
                              <div
                                class="css-1a9yw75"
                                style="width: 0px;"
                              />
                            </div>
                          </div>
                          <div
                            class="resize-triggers"
                          >
                            <div
                              class="expand-trigger"
                            >
                              <div
                                style="width: 1px; height: 1px;"
                              />
                            </div>
                            <div
                              class="contract-trigger"
                            />
                          </div>
                        </div>
                        <span
                          class="css-ajf9ek-getStyles"
                        >
                          <p
                            class="css-17wfq1-bodyStyles"
                          >
                            Total
                          </p>
                          <p
                            class="css-17wfq1-bodyStyles"
                          >
                            $135.00
                          </p>
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
